### PR TITLE
15183 Imported Genesys Web Chat

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#15183

*Description of changes:*
- Imported the new Genesys Web Chat Component.
name request genesys

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -9,6 +9,13 @@
                  :webChatStatusUrl = "window['webChatStatusUrl']"
       />
 
+      <!-- Display the Genesys WebMessage -->
+      <GenesysWebMessage
+        genesysURL="https://apps.cac1.pure.cloud/genesys-bootstrap/genesys.min.js"
+        environmentKey="cac1"
+        deploymentKey="5c012aaa-1f17-4d0b-8da2-bea9c1133d5d"
+      />
```
Comment (severinbeauvais) [inline src/App.vue:17]: As in Create UI, don't hard-code these -- use values from configuration.json.

**Review by severinbeauvais — APPROVED** [review]
I think this is OK... but remember to update the app version.

**Review by severinbeauvais — COMMENTED** [review]
You're also missing the new component import :)

**Comment by JazzarKarim** [comment]
> I think this is OK... but remember to update the app version.

I did update it! From 3.3.8 to 3.3.9. Or do you mean something else?

**Comment by JazzarKarim** [comment]
> You're also missing the new component import :)

Which component are you talking about Sev? 😟

**Comment by severinbeauvais** [comment]
> I did update it! From 3.3.8 to 3.3.9. Or do you mean something else?

> Which component are you talking about Sev? 😟 

Oh, I see... First, I missed the package files. Second, you cheated and didn't run `npm i`, so the shared component wasn't installed (as seen in the check errors above). You should always run `npm i` and never update the package lock manually.

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "3.3.7",
```
Comment (severinbeauvais) [inline package-lock.json:3]: From 7 to 9?

**Comment by JazzarKarim** [comment]
> > I did update it! From 3.3.8 to 3.3.9. Or do you mean something else?
> 
> > Which component are you talking about Sev? 😟
> 
> Oh, I see... First, I missed the package files. Second, you cheated and didn't run `npm i`, so the shared component wasn't installed (as seen in the check errors above). You should always run `npm i` and never update the package lock manually.

I did to that actually first but I reverted the change. I looked at previous commits and I thought that this is how you update after doing something since that's what's everybody was doing. So sorry about that. Good to know.

File: package-lock.json (hunk 2 above)
Comment (JazzarKarim) [reply package-lock.json:3]: There was a mistake from the previous commit. Omid forgot to update the package-lock.json.

Comment (severinbeauvais) [reply package-lock.json:3]: Tsk tsk.

OK, this is fine. Thanks.

**Pull request merged**